lib/const.rb

```diff
@@ -57,3 +57,4 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD}"
+CREW_MESON_OPTIONS = "-Dprefix=#{CREW_PREFIX} -Dlibdir=#{CREW_LIB_PREFIX} -Ddatadir=#{CREW_LIB_PREFIX} -Dmandir=#{CREW_MAN_PREFIX} -Dbuild=#{CREW_BUILD} -Dhost=#{CREW_BUILD} -Dtarget=#{CREW_BUILD} -Dbuildtype=release"
```

Comment:
Nice!  We could probably add `CREW_PYTHON_OPTIONS` as well in the future.  Please bump the crew version at the top of this file.

Reply:
We'll need to add this new constant above line 325 in the `crew` ruby script file as well.

Reply:
Come to think of it, we could also add `CREW_CMAKE_OPTIONS`.

Reply:
I've also been wondering if it would make sense to drop an LDFLAGS option into the OPTIONS lines as well.

e.g.
```
CREW_OPTIONS = "--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --mandir=#{CREW_MAN_PREFIX} --build=#{CREW_BUILD} --host=#{CREW_BUILD} --target=#{CREW_BUILD} LDFLAGS=\"-L#{CREW_LIB_PREFIX} -L/usr/#/{ARCH_LIB} -L/#{ARCH_LIB} -L/usr/lib -L/lib\""
```

Reply:
Constants added to ```crew``` ruby script as well.

Reply:
FYI LDFLAGS just doesn't work there.